### PR TITLE
Android Studio tutorials rewritten

### PR DESCRIPTION
**Pull request by savuor**
Android dev tutorials were written a long time ago.
Let's refresh them by using Android Studio instead of Eclipse and plugins.

### Changes:
* All instructions rewritten to work on Android Studio
* OpenCV4Android tutorial is redundant, removed
* Hello OpenCV sample is simplified, camera is not used

### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [x] I agree to contribute to the project under Apache 2 License.
- [x] To the best of my knowledge, the proposed patch is not based on a code under GPL or another license that is incompatible with OpenCV
- [x] The PR is proposed to the proper branch
- [x] There is a reference to the original bug report and related work
- [x] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [x] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

File: doc/tutorials/introduction/android_binary_package/android_dev_intro.markdown [hunk 1]
```diff
@@ -46,6 +42,64 @@ key topis:
...
+    1. `git clone https://github.com/opencv/opencv`
+    2. `git clone https://github.com/opencv/opencv_contrib`
+    3. `git clone https://github.com/opencv/opencv_extra`
+    Make sure you are on 4.x by calling `git status` in each of the repositories
+
+5. Create a folder for the OpenCV build
+
+6. Substitute your paths to the script below and run it in the console (pay attention to paths):
+    * If you do not need contrib modules remove `--extra_modules_path` and its corresponding path to contrib repo from the arguments
+    * If you need video I/O support, add `--use_media_ndk` to the list of arguments
```
[the remark this answers is not in the document]
Comment (savuor) [reply doc/tutorials/introduction/android_binary_package/android_dev_intro.markdown:79]: Fixed

**Comment by asmorkalov** [comment]
```
/build/precommit_docs/4.x/opencv/doc/tutorials/introduction/android_binary_package/android_dev_intro.markdown:6: warning: unable to resolve reference to `tutorial_O4A_SDK' for \ref command
/build/precommit_docs/4.x/opencv/doc/tutorials/introduction/android_binary_package/dev_with_OCV_on_Android.markdown:4: warning: unable to resolve reference to `tutorial_O4A_SDK' for \ref command
/build/precommit_docs/4.x/opencv/doc/tutorials/introduction/android_binary_package/dev_with_OCV_on_Android.markdown:118: warning: Found unknown command `\Override'
/build/precommit_docs/4.x/opencv/doc/tutorials/introduction/table_of_content_introduction.markdown:24: warning: unable to resolve reference to `tutorial_O4A_SDK' for \ref command
```

**Comment by asmorkalov** [comment]
General notes:

**Comment by asmorkalov** [comment]
```
doc/tutorials/introduction/android_binary_package/android_dev_intro.markdown:200: trailing whitespace.
+savuor@rostislav-laptop:~/Android/Sdk/platform-tools$ ./adb devices 
doc/tutorials/introduction/android_binary_package/android_dev_intro.markdown:204: trailing whitespace.
+savuor@rostislav-laptop:~/Android/Sdk/platform-tools$ 
program finished with exit code 2
```

**Review by asmorkalov — APPROVED** [review]
:+1:

**Pull request merged**